Source/Scene/GlobeSurfaceTile.js

```diff
@@ -318,7 +318,7 @@ define([
 
         // If this tile's terrain and imagery are just upsampled from its parent, mark the tile as
         // upsampled only.  We won't refine a tile if its four children are upsampled only.
-        var isUpsampledOnly = defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
+        var isUpsampledOnly = false;//defined(surfaceTile.terrainData) && surfaceTile.terrainData.wasCreatedByUpsampling();
```

Comment:
A simple mechanism is now implemented in https://github.com/gberaudo/cesium/tree/c2c_patches_pr3_option
@oterral, I am puzzled about this, I also tried simply the Cesium default and it worked fine (terrain requests limited to a list of levels, tiles refined), see https://github.com/gberaudo/cesium/tree/c2c_patches_pr3_disabled

So are you sure the above line is required? Why does it seem to work without it?